x-pack/platform/plugins/shared/fleet/server/routes/cloud_connector/index.ts

```diff
@@ -102,4 +122,139 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
       },
       getCloudConnectorsHandler
     );
+
+  // GET /api/fleet/cloud_connectors/{cloudConnectorId}
+  router.versioned
+    .get({
+      path: CLOUD_CONNECTOR_API_ROUTES.INFO_PATTERN,
+      security: {
+        authz: {
+          requiredPrivileges: [
+            {
+              anyRequired: [
+                FLEET_API_PRIVILEGES.AGENT_POLICIES.READ,
+                FLEET_API_PRIVILEGES.INTEGRATIONS.READ,
+              ],
+            },
+          ],
+        },
+      },
+      summary: 'Get cloud connector',
+      options: {
+        tags: ['oas-tag:Fleet cloud connectors'],
+        availability: {
+          since: '9.2.0',
+          stability: 'experimental',
+        },
+      },
+    })
+    .addVersion(
+      {
+        version: API_VERSIONS.public.v1,
+        validate: {
+          request: GetCloudConnectorRequestSchema,
+          response: {
+            200: {
+              body: () => GetCloudConnectorResponseSchema,
+              description: 'Indicates a successful call.',
+            },
+            400: {
+              body: genericErrorResponse,
+              description: 'Indicates a bad request call.',
```

Comment:
we don't typically add descriptions for the different status codes, and these are not meaningful, so I suggest to remove all of them from this file